.github/workflows/push_dependabot_metadata.yml

```diff
@@ -0,0 +1,18 @@
+name: Send data to Security Alerts
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 10 * * *'
+
+jobs:
+  send-alerts:
+    runs-on: sfdc-hk-ubuntu-latest
```

Comment:
@bchen528 Hi! Since this is a public repo, does this need to use the `pub-hk-*` runners instead (eg `pub-hk-ubuntu-24.04-ip`), according to https://salesforce.quip.com/bu6UA0KImOxJ?